internal/view/app.go

```diff
@@ -691,6 +693,54 @@ func (a *App) helpCmd(evt *tcell.EventKey) *tcell.EventKey {
 	return nil
 }
 
+// previousView returns to the view prior to the current one in the history
+func (a *App) previousView(evt *tcell.EventKey) *tcell.EventKey {
+	if evt != nil && evt.Rune() == rune(ui.KeyB) && a.Prompt().InCmdMode() {
+		return evt
+	}
+	cmds := a.cmdHistory.List()
+	if !(len(cmds) > 1) {
+		dialog.ShowError(
+			a.Styles.Dialog(),
+			a.Content.Pages,
+			"Can't go back any further")
+		return evt
+	} else {
+		previousCmd := cmds[1]
+		a.cmdHistory.Pop()
+		a.gotoResource(previousCmd, "", true)
+		a.ResetCmd()
+	}
+
+	return nil
+}
+
+// lastView switches between the last view and this one a la `cd -`
+func (a *App) lastView(evt *tcell.EventKey) *tcell.EventKey {
```

Comment:
As above these are commands and not views since sub-views are not part of the cmd history.